packages/web/src/components/notification/Notification/ArtistRemixContestEndedNotification.tsx

```diff
@@ -74,7 +74,9 @@ export const ArtistRemixContestEndedNotification = (
         </NotificationBody>
         {isRemixContestWinnersMilestoneEnabled && remixCount > 0 && (
           <Button css={{ width: 'fit-content' }} size='small' asChild>
-            <Link to={pickWinnersRoute}>Pick Winners</Link>
+            <Link to={pickWinnersRoute} onClick={(e) => e.stopPropagation()}>
```

Comment:
also why is the stopPropagation necessary?

Reply:
weirdly when it wasnt there the press would not link to winners page, it would do the onclick from the body which takes us to track page